kedro/config/omegaconf_config.py

```diff
@@ -145,9 +148,17 @@ def __init__(  # noqa: PLR0913
         # Register globals resolver
         self._register_globals_resolver()
 
+        # Store original conf_source
+        self._original_conf_source = conf_source
```

Comment:
Why is this necessary? It's not being used anywhere else after this.

Reply:
Good catch this was left behind.